### PR TITLE
feat: add podAnnotations for all pods

### PR DESCRIPTION
**Pull request by benjamin-texier**
Allows to add annotations on all mender pods.
Like `istio-injection: enabled` or `linkerd.io/inject: disabled`

Changelog: title

Signed-off-by: Benjamin Texier <benjamin.texier@outlook.com>

### CODE REVIEW

File: mender/templates/iot-manager-deploy.yaml [hunk 1]
```diff
@@ -32,6 +32,10 @@ spec:
 
   template:
     metadata:
+      {{- with .Values.azure_iot_manager.podAnnotations }}
```
Comment (alfrunes) [inline mender/templates/iot-manager-deploy.yaml:35]:
```suggestion
      {{- with .Values.iot_manager.podAnnotations }}
```

**Comment by tranchitella** [comment]
@benjamin-texier can you please apply the change suggested by @alfrunes?
Other than that, the PR looks good.

File: mender/templates/iot-manager-deploy.yaml (hunk 1 above)
Comment (benjamin-texier) [reply mender/templates/iot-manager-deploy.yaml]: Thanks @alfrunes

**Comment by benjamin-texier** [comment]
I'm sorry, I was unavailable this week.
In theory, everything is good now

**Review by tranchitella — APPROVED** [review]

**Pull request merged**